lib/src/license_detection/tokenizer.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:collection';
+import 'dart:convert';
+
+class Token {
+  String token;
+  int? position;
+  final int line;
+
+  Token(this.token, this.position, this.line);
+}
+
+class Tokenizer {
+  static const lineSplitter = LineSplitter();
+  static List<Token> tokenize(String text) {
+    var tokens = <Token>[];
+
+    var lines = lineSplitter.convert(text);
```

Comment:
How much do you trust the linesplitter? does it handle CR LF well?

Reply:
Actually while normailizing I replace CR with \n and LineSplitter works well \n so we should not have any problems.